bird_cloud_gnn/radar_dataset.py

```diff
@@ -72,7 +80,7 @@ def __init__(
             name=name,
             hash_key=(
                 name,
-                data_folder,
+                data,
```

Comment:
This is used only for determining the cache name (AFAICT). Se maybe using `data_folder` is better (I don't know if `None` is allowed).

If we introduce `input_file` (see another comment), then we could use `data_folder` or `input_file` here.